Shopipy.AppointmentManagement/Services/AppointmentService.cs

```diff
@@ -0,0 +1,187 @@
+using Shopipy.Persistence.Models;
+using Shopipy.Persistence.Repositories;
+using System.Linq;
+using System.Runtime.InteropServices.JavaScript;
+
+namespace AppointmentManagement.Services;
+
+public class AppointmentService
+{
+    private readonly IGenericRepository<Appointment> _appointmentRepository;
+    private readonly IGenericRepository<Service> _serviceRepository;
+    private readonly IGenericRepository<User> _userRepository;
+
+    public AppointmentService(IGenericRepository<Appointment> appointmentRepository,
+        IGenericRepository<Service> serviceRepository, IGenericRepository<User> userRepository)
+    {
+        _appointmentRepository = appointmentRepository;
```

Comment:
I do not see much point in splitting Appointments and Services into separate projects (separate project = candidate for microservice, appointments are too coupled to services)